packages/wonder-blocks-tabs/src/components/navigation-tab-item.tsx

```diff
@@ -111,10 +111,10 @@ const styles = StyleSheet.create({
         listStyle: "none",
         display: "inline-flex",
         [":has(a:hover)" as any]: {
-            boxShadow: `inset 0 calc(${sizing.size_020}*-1) 0 0 ${semanticColor.action.secondary.progressive.hover.foreground}`,
```

Comment:
This makes sense! I wasn't sure which semantic token to use for the tabs components and the secondary action had the most similar styles!

Will we also be removing the action tokens in favour of the core tokens?